src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $cardWidth;
+  line-height: $lineHeitghtBig;
+  font-size: $fontSizeMiddle;
+  padding: 32px 16px 16px;
+  border: 1px solid $borderColor;
+  border-radius: 5px;
+
+  &__logo {
+    padding: 0 3px;
+  }
+
+  &__img {
+    width: $cardLogoWidth;
+    height: $cardLogoHeight;
+  }
+
+  img {
+    width: $cardLogoWidth;
+    height: $cardLogoHeight;
+  }
+
+  &__description {
+    margin-top: 40px;
+  }
+
+  &__main {
+    font-weight: 500;
+  }
+
+  &__code {
+    margin-top: 4px;
+    color: $addishionalCardColor;
+    font-size: $fontSizeSmall;
+    line-height: $lineHeitghtSmall;
+  }
+
+  &__review {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    font-size: $fontSizeSmall;
+    line-height: $lineHeitghtSmall;
+  }
+
+  &__prices {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price {
+    color: $addishionalCardColor;
+  }
+
+  &__number {
+    font-weight: 700;
+    font-size: $fontSizeUltra;
+    line-height: $lineHeitghtBig;
```

Comment:
There is a typo in the variable name `$lineHeitghtBig`. It should be `$lineHeightBig` to follow the correct spelling of 'height'.